src/components/Bridge/bridge.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import Bridge from '.';
+
+export default {
+  component: Bridge,
+  title: 'Components/Bridge/Bridge',
+  parameters: {
+    docs: {
+      description: {
+        component: 'This is the main component of the bridge page.',
+      },
+    },
+  },
+};
+
+const TemplateBridge: ComponentStory<typeof Bridge> = (args: any) => {
+  return <Bridge {...args} />;
```

Comment:
dropdown text is not visible.

![Screenshot 2022-09-27 at 11 38 20 AM](https://user-images.githubusercontent.com/6604146/192446295-4192b7e8-c97a-4dd7-961a-1af1f118a433.png)

Reply:
Fixed